sites/main-site/src/content/hackathon-projects/hackathon-march-2025/nf-test_migration.md

```diff
@@ -35,9 +35,24 @@ It's a good starting point for first timer to contribute and review Pull-Request
 
 As there are a lot of modules (i.e. 162) and subworkflows (i.e. 23) to migrate we need to stay organised to avoid duplicated work. To do so, we created three issues to track every contribution and a simple procedure as follows:
 
-- Check if no one is already assigned to the modules/subworkflow you want to work on.
+- Check if no one is already assigned to the modules/subworkflow you want to work on using one of the three main issues:
+  - [Modules Batch 1](https://github.com/nf-core/modules/issues/7529)
+  - [Modules Batch 2](https://github.com/nf-core/modules/issues/7654)
+  - [Subworkflows Batch 1](https://github.com/nf-core/modules/issues/7575)
 - Assign yourself to the sub-issue
 - Perform the migration
-- When your pull request is ready, ask for a review and link it to the sub-issue
-- When merged update the hackathon project
+  - Fork [`nf-core/modules`](https://github.com/nf-core/modules/fork) repository to your Github account
+  - Clone locally or use gitpod
+  - Clone [pytest2nf-test](https://github.com/GallVp/pytest2nf-test) `git clone https://github.com/GallVp/pytest2nf-test`
+    - Install it `./gradlew installDist`
+    - Create alias or add to `PATH`: `alias pytest2nf-test=${PWD}/build/install/pytest2nf-test/bin/pytest2nf-test`
+  - Go to modules repo and create a branch `git checkout -b <tool_name>`
+  - Run `pytest2nf-test --nfcore-modules <tool_name>`
+  - Run `nf-core modules test <tool_name> -profile <singularity/docker/conda>` and `nf-core modules lint <tool_name>`
+  - Fix nf-test if necessary
+  - Check if modules follows the [guidelines](https://nf-co.re/docs/tutorials/tests_and_test_data/nf-test_writing_tests)
+  - Commit change and create PR
+- Link to sub-issue : Add `#75XX` in the details
```

Comment:
are you sure about this hash and not using a complete one

Reply:
It's just an example, should i add an image for explanation or just remove it ?

Reply:
I am just not sure what you want to reference here.

Reply:
It's just that when the PR is ready, they should add the sub-issue tag to the PR comment to ensure that the PR and the issue is correctly linked.